### PR TITLE
Update illumina_run_parameter_parser for parsing run stats

### PR DESCRIPTION
**Pull request by chuan-wang**
Tested on LIMS-stage and worked as expected.

### CODE REVIEW

File: scripts/illumina_run_parameter_parser.py [hunk 1]
```diff
@@ -91,6 +93,75 @@ def attach_xml(process, run_dir):
...
+    return run_stats_summary
+
+
+def set_run_stats_in_lims(process, run_stats_summary):
+    for art in process.all_outputs():
+        if 'Lane' in art.name:
+            lane_nbr = int(art.name.split(' ')[1])
+            read = 1
+            for i in run_stats_summary[lane_nbr].keys():
+                if not math.isnan(run_stats_summary[lane_nbr][i]['density']):
```
Comment (aanil) [inline scripts/illumina_run_parameter_parser.py:140]: `run_stats_summary[lane_nbr][i]` assigned to a variable could improve readability as its used a lot of times.

**Review by aanil — APPROVED** [review]
I'm not quite sure of the functionality but the code looks ok to me. I leave it up to you if you want to make the change I suggested.

File: scripts/illumina_run_parameter_parser.py (hunk 1 above)
Comment (chuan-wang) [reply scripts/illumina_run_parameter_parser.py:140]: OK! I will refactor it!

File: scripts/illumina_run_parameter_parser.py [hunk 2]
```diff
@@ -91,6 +93,76 @@ def attach_xml(process, run_dir):
...
+                    art.udf["%PF R{}".format(read)]                       =  stats['percent_pf']
+                if not math.isnan(stats['phasing']):
+                    art.udf["% Phasing R{}".format(read)]                 =  stats['phasing']
+                if not math.isnan(stats['prephasing']):
+                    art.udf["% Prephasing R{}".format(read)]              =  stats['prephasing']
+                if not math.isnan(stats['reads_pf']):
+                    art.udf["Reads PF (M) R{}".format(read)]              =  stats['reads_pf']/1000000
+                if not math.isnan(stats['yield_g']):
+                    art.udf["Yield PF (Gb) R{}".format(read)]             =  stats['yield_g']
+                read += 1
```
Comment (aanil) [inline scripts/illumina_run_parameter_parser.py:161]: Will there always be as many reads as keys in `run_stats_summary[lane_nbr].keys()`?

Comment (chuan-wang) [reply scripts/illumina_run_parameter_parser.py:161]: No, there can be 1 or 2 reads. If there is only 1 read `read += 1` will be run but it doesn't affect anything.

Comment (aanil) [reply scripts/illumina_run_parameter_parser.py:161]: hmm ok

**Pull request merged**